src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -365,6 +431,8 @@ const PlrDaoStakingTransactionBlock = ({
   useEffect(() => {
     // update transaction block with best offer (LiFi)
     const offer = availableOffers?.find((availableOffer) => availableOffer.provider === selectedOffer?.value);
+    const route = availableRoutes?.find((availableRoute) => availableRoute.id === selectedRoute?.value);
```

Comment:
One more point, not that important since `?.` operator does a lot of checks it's better to avoid it , in some cases. I'm attaching an example response here
<img width="1065" alt="Screenshot 2023-02-20 at 8 58 02 PM" src="https://user-images.githubusercontent.com/52369985/220146940-394f8d8d-c80b-49f1-abdb-8d18d5d224d9.png">

Reply:
It's not that important though